Openshift4/operator/artifactory-ha-operator/helm-charts/openshift-artifactory-ha/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: artifactory-ha
   repository: https://charts.jfrog.io/
-  version: 107.77.7
+  version: 107.77.10
 digest: sha256:4bd55f5c331c175276c928d9b4187bff9e91a35f5ec787d5e69e8926986bfbf7
```

Comment:
digest value is not changed. make sure you run a helm dependency update so that the value also gets updated.